apps/app/src/pages/Build/ScreenshotsDiffCard.js

```diff
@@ -95,7 +95,7 @@ export function ScreenshotsDiffCard({
             as={ScreenshotDiffStatusIcon(screenshotDiff.status)}
             color={getStatusPrimaryColor(screenshotDiff.status)}
           />
-          {compareScreenshot.name || baseScreenshot.name}
+          {compareScreenshot?.name || baseScreenshot.name}
```

Comment:
Just leave base screenshot?

Reply:
If we use only base screenshot name, there will be a bug on « added » Diff. This component is used in every screenshot sections

Reply:
Got it!